invision-forum-crawler.py

```diff
@@ -776,7 +797,8 @@ def main():
 
     try:
         file_name_temp_zst, total_docs_archive, total_docs_len = merge_archive()
-        new_name_zst = datetime.datetime.today().strftime('%Y%m%d%H%M%S') + '-' + file_name_zst
+        # new_name_zst = datetime.datetime.today().strftime('%Y%m%d%H%M%S') + '-' + file_name_zst
+        new_name_zst = file_name_zst # Archive name should not be enchanced with timestamp as it'll need additional actions
```

Comment:
The timestamp added earlier was to distinguish the files and check their correctness
Now there is no need for that